sites/next.skeleton.dev/src/content/docs/components/tabs/svelte.mdx

```diff
@@ -84,3 +83,7 @@ Set `fluid` to enable tabs to stretch to fill the avialable width.
 		<Code code={ExampleRTLRaw} lang="svelte" />
 	</Fragment>
 </Preview>
+
+## API Reference
+
+<ApiTable types={['TabsRootProps', 'TabsListProps', 'TabsControlProps', 'TabsContentProps', 'TabsPanelProps']} schema={TabSchema} />
```

Comment:
For some reason only the TabsControl and TabsPanel types are rendered as tables. The rest are missing.

Reply:
Fixed aswell, just name things